services/src/main/java/io/druid/cli/validate/DruidJsonValidator.java

```diff
@@ -73,7 +86,23 @@ public void run()
       System.out.printf("File[%s] does not exist.%n", file);
     }
 
-    final ObjectMapper jsonMapper = new DefaultObjectMapper();
+    final Injector injector = GuiceInjectors.makeStartupInjector();
```

Comment:
@drcrallen I've tried but couldn't make it initialized properly by exception `No implementation for java.lang.String annotated with @com.google.inject.name.Named(value=serviceName) was bound.` or `Exception in thread "main" java.lang.IllegalArgumentException: Unrecognized inject value id type (com.google.inject.Key), expecting String`. Could you do that?